src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf");
+}
+
+.label1 {
+  position: relative;
+}
+
+.inp1 {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.label2 {
+  position: relative;
+}
+
+.inp2 {
+  width: 100%;
+  height: 42px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.img2 {
+  position: absolute;
+  top: 4px;
+  left: 26px;
+  width: 11px;
+  height: 11px;
+}
+
+.img1 {
+  position: absolute;
+  top: 0;
+  left: 26px;
+}
+
+.inp:default {
```

```suggestion
.inp {
```
but use full names ~inp~ => `input`